source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseActionElement.mm

```diff
@@ -189,4 +191,38 @@ + (NSNumber *)getKey:(ACRActionType)actionType
     return key;
 }
 
+- (NSString *)tooltip
+{
+    if (_tooltip && _tooltip.length) {
+        return _tooltip;
+    }
+
+    if (_elem && !_elem->GetTooltip().empty()) {
+        _tooltip = [NSString stringWithCString:_elem->GetTooltip().c_str() encoding:NSUTF8StringEncoding];
+        return _tooltip;
+    }
+
+    return nil;
+}
+
+- (void)setTooltip:(NSString *)tooltip
+{
+    if (tooltip) {
```

Comment:
why is the tooltip string emptiness not verified as in line 200?

Reply:
It is verified. _tooltip is the property name accessed within an object.